### PR TITLE
Add Ceilidh Welsh to Turing Way contributors.md

### PR DESCRIPTION
**Pull request by CeilidhWelsh**
### Summary

Adding Ceilidh Welsh, related to https://github.com/the-turing-way/the-turing-way/pull/3514

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?
- [ ] Typos 
- [ ] Anything that might be missing 

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -482,6 +482,21 @@ I also hope that it can convince people who don't consider themselves capable of
...
+### Ceilidh Welsh
+
+* Role: Book Dash Participant (2023)
+* GitHub id: [CeilidhWelsh](https://github.com/CeilidhWelsh)
+* ORCID: [0000-0002-7332-7392](https://orcid.org/my-orcid?orcid=0000-0002-7332-7392)
+* Short bio:
+> Ceilidh is a PhD student in the Department of Oncology at the University of Cambridge, funded by Cancer Research UK RadNet. She investigates predictive biomarkers for radiotherapy response in head-and-neck cancer. She was awarded the Alan Turing Institute Enrichment Scheme Award for 2022/2023, where she explored her interests in data ethics, data reproducibility and community-led data science projects. 
+
+* Personal highlights:
+> Learning about the Turing Way project and how community-led science can produce and sustain projects that bring people together.
```
Comment (aleesteele) [inline contributors.md:494]:
```suggestion
> Learning about _The Turing Way_ project and how community-led science can produce and sustain projects that bring people together. 
```

File: contributors.md [hunk 2]
```diff
@@ -482,6 +482,21 @@ I also hope that it can convince people who don't consider themselves capable of
...
+
+* Role: Book Dash Participant (2023)
+* GitHub id: [CeilidhWelsh](https://github.com/CeilidhWelsh)
+* ORCID: [0000-0002-7332-7392](https://orcid.org/my-orcid?orcid=0000-0002-7332-7392)
+* Short bio:
+> Ceilidh is a PhD student in the Department of Oncology at the University of Cambridge, funded by Cancer Research UK RadNet. She investigates predictive biomarkers for radiotherapy response in head-and-neck cancer. She was awarded the Alan Turing Institute Enrichment Scheme Award for 2022/2023, where she explored her interests in data ethics, data reproducibility and community-led data science projects. 
+
+* Personal highlights:
+> Learning about the Turing Way project and how community-led science can produce and sustain projects that bring people together. 
+> Co-authoring the [Data Hazards Chapter]() with [Susana Roman Garcia](https://github.com/Susana465), [Natalie Zelenka](https://github.com/NatalieZelenka), Irma Hafidz and many more incredible people doing incredible things that helped us along the way!
```
Comment (aleesteele) [inline contributors.md:495]: I think this is missing a link! You can use our internal reference syntax to do this: https://the-turing-way.netlify.app/community-handbook/style/style-crossref

Comment (CeilidhWelsh) [reply contributors.md]: I think that has been corrected now!

**Review by aleesteele — APPROVED** [review]
Approved, thanks Ceilidh! So glad you're finally in the book!

**Pull request merged**